modules/va_forms/app/workers/va_forms/form_reloader.rb

```diff
@@ -1,31 +1,23 @@
 # frozen_string_literal: true
 
 require 'sidekiq'
-require 'va_forms/regex_helper'
 
 module VAForms
   class FormReloader
     include Sidekiq::Worker
-    include SentryLogging
-    FORM_BASE_URL = 'https://www.va.gov'
+
+    # Only retry for ~30 minutes since the job that spawns this one runs every hour
```

Comment:
This comment isn't accurate, is it? I thought the `FormReloader` ran nightly.

Reply:
You are right. I should have ganked this line out anyway when I updated the `retry` param.